fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/spring/ModeShapeRepositoryFactoryBean.java

```diff
@@ -151,4 +153,27 @@ private DefaultPropertiesLoader getPropertiesLoader() {
         }
         return propertiesLoader;
     }
+
+    /**
+     * Set the mode for server managed properties.  The value "strict" (default)
+     * will result in the classic behavior where certain provenance-related
+     * properties cannot be set, while "relaxed" will allow for them to be
+     * set in some circumstances.
+     *
+     * @param value the value (must be either "strict" or "relaxed")
+     */
+    public void setServerManagedPropertiesMode(final String value) {
```

Comment:
There were two reasons for this approach.  
1.  the shared tooling to test whether something was a server managed property had been done at a static scope (and shared between modules).  Whether something is a "relaxed server managed property" seemed to merit a similar scope (and was much easier to implement in such a way).  

2. My first set of integration tests involved not setting the system property (because, setting a system property during testing is a terrible idea that will forever prevent parallel testing and so forth)... but I couldn't get one integration test using the spring runner to shut down (and free up the ports) before the other started.  While I could have had separate ports for the different sprint configurations it was getting awfully messy and the compromise of using a system property made more sense.

Finally, when running fedora in production, the spring config files are relatively inaccessible and it's preferable to just use a -D flag (system property) as part of your tomcat environment anyway.